### PR TITLE
Release v4.7.0-rc1

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

**Review by daira — APPROVED** [review]
utACK. There should be a `man` page for `zcashd-wallet-tool` but that is not blocking for the RC. I'll add it to #5713.

**Review by steven-ecc — APPROVED** [review]
utACK

File: contrib/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+zcash (4.7.0~rc1) stable; urgency=medium
```
Comment (LarryRuane) [inline contrib/debian/changelog:1]:
```suggestion
zcash (4.7.0-rc1) stable; urgency=medium
```

File: doc/release-notes/release-notes-4.7.0-rc1.md [hunk 2]
```diff
@@ -0,0 +1,538 @@
...
+Zcashd release.
+
+Following the upgrade to 4.7.0, Zcashd will require that the user confirm that
+they have backed up their new emergency recovery phrase, which may be obtained
+from the output of the `z_exportwallet` RPC call. This confirmation can be
+performed manually using the `zcashd-wallet-tool` utility that is supplied
+with this release (built or installed in the same directory as `zcashd`).
+The wallet will not allow the generation of new addresses until this
+confirmation has been performed. It is recommended that after this upgrade,
+that funds tied to preexisting addresses be migrated to newly generated
```
Comment (LarryRuane) [inline doc/release-notes/release-notes-4.7.0-rc1.md:23]:
```suggestion
funds tied to preexisting addresses be migrated to newly generated
```

File: doc/release-notes/release-notes-4.7.0-rc1.md [hunk 3]
```diff
@@ -0,0 +1,538 @@
...
+recovery phrase going forward. If you choose not to migrate funds in this
+fashion, you will continue to need to securely back up the entire `wallet.dat`
+file to ensure that you do not lose access to existing funds; EXISTING FUNDS
+WILL NOT BE RECOVERABLE USING THE EMERGENCY RECOVERY PHRASE UNLESS THEY HAVE
+BEEN MOVED TO A NEWLY GENERATED ADDRESS FOLLOWING THE 4.7.0 UPGRADE.
+
+New RPC Methods
+---------------
+
+- 'walletconfirmbackup' This newly created API checks a provided emergency
```
Comment (LarryRuane) [inline doc/release-notes/release-notes-4.7.0-rc1.md:34]:
```suggestion
- `walletconfirmbackup` This new API checks a provided emergency
```

File: doc/release-notes/release-notes-4.7.0-rc1.md [hunk 4]
```diff
@@ -0,0 +1,538 @@
...
+  `-requirewalletbackup=false` but this is not recommended unless you know what
+  you're doing (and have otherwise backed up the wallet's recovery phrase anyway).
+  For security reasons, this RPC method is not intended for use via zcash-cli 
+  but is provided to enable `zcashd-wallet-tool` and other third-party wallet 
+  interfaces to satisfy the backup confirmation requirement. Use of the 
+  `walletconfirmbackup` API via zcash-cli would risk that the recovery phrase 
+  being confirmed might be leaked via the user's shell history or the system
+  process table; `zcashd-wallet-tool` is specifically provided to avoid this
+  problem.
+- 'z_getbalanceforviewingkey' This newly created API allows a user to obtain
```
Comment (LarryRuane) [inline doc/release-notes/release-notes-4.7.0-rc1.md:47]:
```suggestion
- `z_getbalanceforviewingkey` This new API allows a user to obtain
```

File: doc/release-notes/release-notes-4.7.0-rc1.md [hunk 5]
```diff
@@ -0,0 +1,538 @@
...
+  control what kind of information they permit `zcashd` to reveal when creating
+  the transaction. If the transaction can only be created by revealing more
+  information than the given strategy permits, `z_sendmany` will return an
+  error. The parameter defaults to `LegacyCompat`, which applies the most
+  restrictive strategy `FullPrivacy` when a Unified Address is present as the
+  sender or a recipient, and otherwise preserves existing behaviour (which
+  corresponds to the `AllowFullyTransparent` policy).
+
+- Since Sprout outputs are no longer created (with the exception of change)
+  'z_sendmany' no longer generates payment disclosures (which were only
```
Comment (LarryRuane) [inline doc/release-notes/release-notes-4.7.0-rc1.md:92]:
```suggestion
  `z_sendmany` no longer generates payment disclosures (which were only 
```

File: doc/release-notes/release-notes-4.7.0-rc1.md [hunk 6]
```diff
@@ -0,0 +1,538 @@
...
+WILL NOT BE RECOVERABLE USING THE EMERGENCY RECOVERY PHRASE UNLESS THEY HAVE
+BEEN MOVED TO A NEWLY GENERATED ADDRESS FOLLOWING THE 4.7.0 UPGRADE.
+
+New RPC Methods
+---------------
+
+- 'walletconfirmbackup' This newly created API checks a provided emergency
+  recovery phrase against the wallet's emergency recovery phrase; if the phrases
+  match then it updates the wallet state to allow the generation of new addresses.
+  This backup confirmation workflow can be disabled by starting zcashd with
```
Comment (LarryRuane) [inline doc/release-notes/release-notes-4.7.0-rc1.md:37]:
```suggestion
  This backup confirmation workflow can be disabled by starting `zcashd` with 
```

File: doc/release-notes/release-notes-4.7.0-rc1.md [hunk 7]
```diff
@@ -0,0 +1,538 @@
...
+New RPC Methods
+---------------
+
+- 'walletconfirmbackup' This newly created API checks a provided emergency
+  recovery phrase against the wallet's emergency recovery phrase; if the phrases
+  match then it updates the wallet state to allow the generation of new addresses.
+  This backup confirmation workflow can be disabled by starting zcashd with 
+  `-requirewalletbackup=false` but this is not recommended unless you know what
+  you're doing (and have otherwise backed up the wallet's recovery phrase anyway).
+  For security reasons, this RPC method is not intended for use via zcash-cli
```
Comment (LarryRuane) [inline doc/release-notes/release-notes-4.7.0-rc1.md:40]:
```suggestion
  For security reasons, this RPC method is not intended for use via `zcash-cli`
```

File: doc/release-notes/release-notes-4.7.0-rc1.md [hunk 8]
```diff
@@ -0,0 +1,538 @@
...
+- 'walletconfirmbackup' This newly created API checks a provided emergency
+  recovery phrase against the wallet's emergency recovery phrase; if the phrases
+  match then it updates the wallet state to allow the generation of new addresses.
+  This backup confirmation workflow can be disabled by starting zcashd with 
+  `-requirewalletbackup=false` but this is not recommended unless you know what
+  you're doing (and have otherwise backed up the wallet's recovery phrase anyway).
+  For security reasons, this RPC method is not intended for use via zcash-cli 
+  but is provided to enable `zcashd-wallet-tool` and other third-party wallet 
+  interfaces to satisfy the backup confirmation requirement. Use of the 
+  `walletconfirmbackup` API via zcash-cli would risk that the recovery phrase
```
Comment (LarryRuane) [inline doc/release-notes/release-notes-4.7.0-rc1.md:43]:
```suggestion
  `walletconfirmbackup` API via `zcash-cli` would risk that the recovery phrase 
```

File: doc/release-notes/release-notes-4.7.0-rc1.md [hunk 9]
```diff
@@ -0,0 +1,538 @@
...
+  `mnemonic_seedfp` and `legacy_seedfp`, the latter of which replaces the field that
+  was previously named `seedfp`. 
+
+Wallet
+------
+
+'z_sendmany'
+------------
+
+- The 'z_sendmany' RPC call no longer permits Sprout recipients in the
```
Comment (LarryRuane) [inline doc/release-notes/release-notes-4.7.0-rc1.md:70]:
```suggestion
- The `z_sendmany` RPC call no longer permits Sprout recipients in the 
```

File: doc/release-notes/release-notes-4.7.0-rc1.md [hunk 10]
```diff
@@ -0,0 +1,538 @@
...
+  exported data.
+
+- The results of the 'getwalletinfo' RPC have been modified to return two new fields:
+  `mnemonic_seedfp` and `legacy_seedfp`, the latter of which replaces the field that
+  was previously named `seedfp`. 
+
+Wallet
+------
+
+'z_sendmany'
```
Comment (LarryRuane) [inline doc/release-notes/release-notes-4.7.0-rc1.md:67]:
```suggestion
`z_sendmany`
```

**Review by LarryRuane — COMMENTED** [review]
utACK 1f3c145e4abc87b123911938888e244aeca9c14d with typo and grammar suggestions

I verified that the changes here correspond to the changes implementing 4.6.0-rc1 (75b539400c7f07e29a34c2ebfbe0a9c1aa8dbf3c, d4b850d3f7579c167177b69472143b9b5268b163, 1c910cf9ccb1c1b5b9083af25bec380873c5b7f1)

File: contrib/debian/changelog (hunk 1 above)
Comment (str4d) [reply contrib/debian/changelog:1]: This is how the Debian tooling generates it; I'm not messing with their semantics.

File: doc/release-notes/release-notes-4.7.0-rc1.md (hunk 2 above)
Comment (str4d) [reply doc/release-notes/release-notes-4.7.0-rc1.md:23]: This can be fixed in the release notes updates before 4.7.0.

**Pull request merged**